extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayRecorder.java

```diff
@@ -27,9 +28,9 @@ public class FlywayRecorder {
 
     static final List<FlywayContainer> FLYWAY_CONTAINERS = new ArrayList<>(2);
 
-    private final FlywayRuntimeConfig config;
+    private final RuntimeValue<FlywayRuntimeConfig> config;
```

Comment:
Hmmm. This is weird? I don't think you need to wrap this to inject the runtime config. At least I don't remember having to do it.

@geoand do I miss something?

Reply:
AFAIR, it does need to be wrapped

Reply:
In the Quarkus codebase I've encountered both styles. So, I initially went without wrapping but encountered errors as soon as I added integration tests.

Reply:
Yeah, I think it depends on timinng, but in general this pattern should be used